README.md

```diff
@@ -18,21 +18,58 @@ Plotly.js can be used to produce dozens of chart types and visualizations, inclu
 
 ## Table of contents
 
-* [Quick start options](#quick-start-options)
+* [Load from CDN](#load-from-cdn)
+* [Load from npm](#load-from-npm)
+* [Versioning](#versioning)
 * [Partial bundles](#partial-bundles)
 * [Alternative ways to require or build plotly.js](#alternative-ways-to-require-or-build-plotlyjs)
 * [Bugs and feature requests](#bugs-and-feature-requests)
 * [Documentation](#documentation)
 * [Contributing](#contributing)
 * [Community](#community)
-* [Versioning](#versioning)
 * [Notable contributors](#creators)
 * [Copyright and license](#copyright-and-license)
 
 ---
-## Quick start options
+## Load from CDN
+Fastly supports Plotly.js with free CDN service. Read more at <https://www.fastly.com/open-source>.
+
+### Minified plotly.js X.Y.Z bundles
+```html
+<head>
+    <script src="https://cdn.plot.ly/plotly-2.0.0-rc.2.min.js"></script>
+</head>
+<body>
+    <div id="gd"></div>
+
+    <script>
+        Plotly.newPlot("gd", {
+            "data": [{
+                "y": [1, 2, 3]
+            }],
+            "layout": {
+                "width": 600,
+                "height": 400
+            }
+        });
+    </script>
+</body>
+```
+In the example above `Plotly` object is added to the window scope by the script in the `head` section.
+The `newPlot` method is then used to draw an interactive figure as described by `data` and `layout` into the desired `div` here named `gd`.
+As demonstrated in the example above basic knowledge of `html` and [JSON](https://en.wikipedia.org/wiki/JSON) syntax is enough to get started i.e. with/without JavaScript!
+To learn and build more with plotly.js please visit [plotly.js documentation](https://plotly.com/javascript).
+
+### Un-minified versions are also available on CDN
+While non-minified source files may contain characters outside UTF-8, it is recommended that you specify the `charset` when loading those bundles.
+```html
+    <script src="https://cdn.plot.ly/plotly-2.0.0-rc.2.js" charset="utf-8"></script>
+```
+
+#### Please note that as of v2 the "plotly-latest" outputs (e.g. https://cdn.plot.ly/plotly-latest.min.js) will no longer be updated on the CDN, and will stay at the last v1 patch v1.58.4. Therefore, to use the CDN with plotly.js v2 and higher, you must specify an exact plotly.js version.
```

Comment:
for little "notes" like this, please don't use `####` but rather prefix them with `>` so they get indented/dimmed a bit.

Reply:
Done in https://github.com/plotly/plotly.js/pull/5705/commits/a0bd25835c3f415dece9c5cd8604663a60d7f532.